swabbie-core/src/main/kotlin/com/netflix/spinnaker/swabbie/events/ResourceStateManager.kt

```diff
@@ -162,37 +163,68 @@ class ResourceStateManager(
     if (event is OptOutResourceEvent) {
       log.debug("Tagging resource ${event.markedResource.uniqueId()} with \"expiration_time\":\"never\"")
```

Comment:
we could probably pull both these logs into the `tagResource(..)` function, and we wouldn't need to return a value from the function. That would make the `tagResource(..)` function a bit more readable.

Reply:
updated `tagResource` function to handle these comments